pipeline/errors/error.go

```diff
@@ -35,12 +19,8 @@ type DeprecationErrorData struct {
 	Docs  string `json:"docs"`
 }
 
-func (e *PipelineError) Error() string {
-	return fmt.Sprintf("[%s] %s", e.Type, e.Message)
-}
-
-func (e *PipelineError) GetLinterData() *LinterErrorData {
-	if e.Type != PipelineErrorTypeLinter {
+func GetLinterData(e *types.PipelineError) *LinterErrorData {
```

Comment:
This func always returns nil. Something does not seem correct eve thought it wan't introduced from this PR.